### ISSUE TITLE
fix for the nav

### ISSUE THREAD

**Issue by Haroenv** [bullgit/bullgit.github.io#25]
```css
.site--navigation .nav--list-item {
    display: inline-block;
    margin-left: 1.4em;
}
```

instead of:

```css
.site--navigation .nav--list-item {
    display: inline-block;
    margin-left: 2.5em;
}
```

**Comment by dervondenbergen** [comment]
this has to be fixed, it is not closed

**Comment by dervondenbergen** [comment]
altough #33 fixes the menu, i think that we should remove compass as a dependencie and replace it with regular sass, that way we could compile the code also with node-sass, which works basically everywhere

**Comment by Haroenv** [comment]
agreed, and the next thing should be to create a gulp (/grunt) task

**Comment by LukyVj** [comment]
t

**Comment by Haroenv** [comment]
closed in #34
